### DISCUSSION TITLE
Welcome to synphage Discussions!

### DISCUSSION THREAD

**Discussion by vestalisvirginis** [vestalisvirginis/synphage#16] [Announcements]
👋 Welcome!

We’re using Discussions as a place to connect with other members of our community. We hope that you:

- Ask questions you’re wondering about;
- Share ideas;
- Engage with other community members;
- Welcome others and are open-minded. Remember that this is a community we
build together 💪.

To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mikeds211** [comment]
My impression. Using a macbook pro M2, OS sequoia 15.0. Installation using conda failed first because dagster complained pkg-config was not present, so I installed that. Then it failed because cairo was no present, so I installed that. Then dagster got up OK and I could go through the many steps required to get to a similarity plot. The blast comparisons of relatively small virus genomes (12-20 kb), and relatively few genomes (26) took much longer than I expected, as in my experience, once the databases have been set up for a blastn or blastp run, then searching such a small database is usually lightning fast. In any case, I finally get to the plot stage and the default format output is far less impressive than say clinker (https://github.com/gamcil/clinker), which gives high quality gene maps and similarity connectors. In clinker I can use an easy interface to alter the output but in synphage I have to play with a config file and run again. For all the extra time needed to set up synphage, and to run the various stages, I am wondering what are the advantages over existing tools, and how can it be made easier and faster to use.

**Reply by canimus** [reply]
Hi @mikeds211 as a contributor for this project, I found your first experience with `synphage` unfortunate. All software suffers some sort of setup burden in new versions of MacOS. I was really interested to look into `clinker` sounds like an interesting project, yet still part of the CLI-type tools that obfuscate processing bioinformatic pipelines, and somehow expedite the process without user intervention, however not surfacing the computation graph nor harnessing the quality of the elephant of the room, which is the data in use. Now, because I consider that every head is a different world, and follow the idea of not judging a book by the cover, I invite you to experience further the use of a DAG, a quality assessment tool, and plotting tool, in a GUI interface like `synphage` and if you would like to contribute to the project, to build on please open a PR and make suggestions that can enrich the ecosystem. At the end of the day, the marginal utility for an ecosystem of tools, will help the entire community, technical or not.
I could enumerate the list of benefits behind a transparent computation pipeline, that gets you from blast query to a visualization, but I believe the post you share was not engaging in that capacity, and more a marketing initiative. Wish you a very nice day.
